home.html

```diff
@@ -16,5 +17,9 @@ <h1>About Git</h1>
         <a href="#">Twitter</a>
         <a href="#">About</a>
         <a href="#">My link</a>
+        <a href="#">Google</a>
```

Comment:
I think Twitter needs to be added as well.

Reply:
Twitter is not required